qtranslate.php

```diff
@@ -54,7 +54,7 @@
 	ja by Brian Parker
 	nl by RobV
 	pt by netolazaro
-	pt-br by Pedro Mendonça
+	pt-pt by Pedro Mendonça
```

Comment:
Let me explain a bit better.

I didn't introduce pt-br, it already exists. 
Brazilian Portuguese already existed in po/mo files, imported from old plugins, and was already present in 'qtranslate.php' but was incorrectly referenced as simply 'pt'.

PT generically is portuguese, pt_PT is from Portugal and pt_BR is the Brazilian Portuguese. Like English (en_EN and en_US).
In the beggining you had just 'pt' and it was a bit mixed up. In the absense of _PT our _BR, it was assumed to be pt-br, it could be easly verified by the in the original brazilian flag ('br.png') and in some the text used in the 'not available message'. This usually hapens when brazilian translation is made first and one assumes it as generic portuguese.

To avoid confusion it sould be clearly separated by pt_PT and pt_BR codes. WordPress use these two codes but also recognize just 'pt' as european portuguese. This is a bit confusing but you can check in the list here: http://wpcentral.io/internationalization/
According *Locale Codes* the language is defined by 'pt' (Portugal) and 'pt-br' (Brazil)
According *WordPress Locale* the language is defined by 'pt_PT' (Portugal) and 'pt_BR' (Brazil)

So, I've started to populate the 'pt' with native portuguese stuff (flag and texts) and incorrectly erased the already existing brazilian ones in this commit https://github.com/qTranslate-Team/qtranslate-x/commit/66d811070a995d3d7de77ffb05e4a0c1fb85cecf

In this second commit https://github.com/qTranslate-Team/qtranslate-x/commit/4673cc5746d61f7bd63ab84cec779463d1321b70 I've reinserted the already existing brazilian stuff but with the correct 'pt-br' code. Looking just at this commit it might look like I've introduced it, but was just a fair reposition of the pre existent with correct code.

In resume, I've renamed the already existing to pt-br and introduced the pt(_PT) european portuguese language that I've been contributing.

I though that it could generate this confusion, but if you check the diff of both commits together you can see that I've maintained the old 'pt' and renamed it correctly to 'pt-br'.
https://github.com/qTranslate-Team/qtranslate-x/commit/d411df985ea0d4509cc25da47c3bfc60db80c0f2

I'm sorry that I created this confusion, but it was a mess already and it needed some clarification.

Thanks